Optimize abstracts & prereqs for containers (CQA) (#5004)
theforeman/foreman-documentation: guides/common/modules/con_importing-container-image-repositories.adoc

Reviewer: sync -> synchronize in line 8+11

[role="_abstract"]
ifndef::orcharhino[]

> Import container image repositories into {Project} from Red Hat Registry or other image registries to sync external images and make them available through content views.

Reviewer: ```suggestion
Import container image repositories into {Project} from Red{nbsp}Hat Registry or other image registries to sync external images and make them available through content views.
```

Author: ```suggestion
Import container image repositories into {Project} from Red{nbsp}Hat Registry or other image registries to synchronize external images and make them available through content views.
```

endif::[]
ifdef::orcharhino[]

> Import container image repositories into {Project} from any container image registry to sync external images and make them available through content views.

Author: ```suggestion
Import container image repositories into {Project} from any container image registry to synchronize external images and make them available through content views.
```
